.gitignore

```diff
@@ -1 +1,2 @@
 node_modules/
+.idea
```

Comment:
Use your global .gitignore for this kind of thing, it's not specific to the project